pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>io.confluent</groupId>
     <artifactId>resolver-maven-plugin</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
```

Comment:
you can still use 1.0.0-SNAPSHOT here, and when you feel ready to release then change to 0.2.0, but this is not a blocker.

Reply:
set to 1.0.0-SNAPSHOT